ccloudvm/kata-pkgs-fedora27.yaml

```diff
@@ -0,0 +1,84 @@
+---
+base_image_url: https://mirror.us-midwest-1.nexcess.net/fedora/releases/27/CloudImages/x86_64/images/Fedora-Cloud-Base-27-1.6.x86_64.qcow2
```

Comment:
This URL doesn't look that "official" wrt the Fedora project - could you maybe find a fedoraproject one instead?